### ISSUE TITLE
Pynput is inserting < (less-than sign) before first character typed (Monitoring)

### ISSUE THREAD

**Issue by danielkovarik** [moses-palmer/pynput#285]
Hello,

I noticed, that when using `US (India)` keyboard layout, pynput inserts `<` (less-than sign) before the first character I type during **monitoring**. 

Is there anything I can do to avoid this weird behavior?

For better clarity, I am attaching a video of the issue.

OS: Windows 10
pynput version: 1.6.8
keyboard layout: US (India) keyboard
![pynput-india](https://user-images.githubusercontent.com/35225229/87313942-e339d700-c522-11ea-9678-80b793afdf53.gif)

**Comment by moses-palmer** [comment]
Thank you for your report.

This is indeed strange---so strange, in fact, that I doubted that I would be able to reproduce it. It appears to be caused by calls to [`ToUnicodeEx`](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-tounicodeex) when generating the keyboard layout.

I have pushed a small modification [here](https://github.com/moses-palmer/pynput/tree/fixup-win32-keyboard-listener) which I invite you to test; it works in my limited testing.
